samigo/samigo-app/src/webapp/jsf/delivery/beginTakingAssessment.jsp

```diff
@@ -55,12 +55,19 @@
 <!-- DONE BUTTON FOR PREVIEW -->
 <h:panelGroup rendered="#{delivery.actionString=='previewAssessment'}">
   <div class="sak-banner-info">
-     <h:outputText value="#{deliveryMessages.ass_preview}" escape="false" />
+     <h:outputText value="#{deliveryMessages.ass_preview}" escape="true" />
+     <br>
+     <h:outputText value="#{deliveryMessages.seb_preview_info}" escape="false"/>
+     <br>
+     <br>
      <h:commandButton value="#{deliveryMessages.exit_preview}"
-        styleClass="exitPreviewButton"
-        action="#{person.cleanResourceIdListInPreview}"
-        type="submit"
-        onclick="return returnToHostUrl(\"#{delivery.selectURL}\");" />
+     styleClass="exitPreviewButton"
+     action="#{person.cleanResourceIdListInPreview}"
+     type="submit"
+     onclick="return returnToHostUrl(\'#{delivery.selectURL}\'');" />
+    </div>
```

Comment:
these extra divs?

Reply:
yeah, looks like the closing div is too much